### ISSUE TITLE
Pretty-printed square roots are misaligned with combining unicode characters

### ISSUE THREAD

**Issue by TomFryers** [sympy/sympy#25503]
```python
>>> from sympy import *
>>> pprint(sqrt(Symbol("x") + 1) + sqrt(Symbol("x")**2))  # OK
               ____
  _______     ╱  2 
╲╱ x + 1  + ╲╱  x  
>>> pprint(sqrt(Symbol("xbar") + 1) + sqrt(Symbol("x")**2))  # Bad
               ____
  _______     ╱  2 
╲╱ x̅ + 1 + ╲╱  x  
```
As you can hopefully see, although the GitHub rendering doesn't seem particularly clear, the final square root is aligned incorrectly.

**Comment by sydneyco** [comment]
I don't seem to have this problem when I run it. Has this task been resolved?

**Comment by oscarbenjamin** [comment]
This was fixed in 539b71ef3560f951e32f2724beb97af6d01ee040 from gh-25673.
